index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" content"OOP test. Make use of constructor functions to create a fun Pirate website.">
+    <title>Pirate Bartender</title>
+    <link rel="stylesheet" href="main.css">
+    <script src="app.js"></script>
+    <script src="https://code.jquery.com/jquery-3.1.0.min.js"   integrity="sha256-cCueBR6CsyA4/9szpPfrX3s49M9vUU5BgtiJj06wt/s="   crossorigin="anonymous"></script>
```

Comment:
For better performance, it is better if you put the "<script>" elements at the end, just before "</body>". Also, you have a double space between "src" and "integrity" on line 8.